scripts/arm

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# DESC: Build the Harvester container without running tests
```

Comment:
This is weird. From the filename itself, I was assuming this to be related to the arm architecture because I have never looked at the file.

Reply:
Maybe the solution here should be change the file, because looks what is inside:

`set -e

cd $(dirname $0)


./build
./package-webhook
./package-upgrade
./package`

WDYT @ihcsim

Reply:
This is used in CI:

```
.github/workflows/build.yml:31:    - name: Run make arm   
```

Reply:
This files runs the build for ARM Architecture. Thanks guys.